### PR TITLE
docs: manual create entities on hass

### PR DESCRIPTION
**Pull request by fanningert**
Add example to execute scene from home assistant

### CODE REVIEW

**Comment by fanningert** [comment]
https://github.com/zwave-js/zwavejs2mqtt/issues/387

File: docs/guide/homeassistant.md [hunk 1]
```diff
@@ -231,6 +231,21 @@ The main template is like the thermostat template. The things to add are:
 }
 ```
 
+## Execute API functions
+
+### Execute scene
+
+In this example I will execute the scene with the id `1` via a switch, with following entries in the `configuration.yaml`. Replace `<mqtt_prefix>` and `<mqtt_name>` with you values.
```
Comment (chilicheech) [inline docs/guide/homeassistant.md:238]: `with your values`

Comment (fanningert) [reply docs/guide/homeassistant.md:238]: A better variant?
```
In this example I will execute the scene with the id `1` via a switch. 
Replace `<mqtt_prefix>` and `<mqtt_name>` with you values and add this lines in the `configuration.yaml` of home assistant.
```

File: docs/guide/homeassistant.md [hunk 2]
```diff
@@ -231,6 +231,21 @@ The main template is like the thermostat template. The things to add are:
 }
 ```
 
+## Execute API functions
+
+### Execute scene
+
+In this example I will execute the scene with the id `1` via a switch, with the following entries in the `configuration.yaml`. Replace `<mqtt_prefix>` and `<mqtt_name>` with your values.
```
Comment (robertsLando) [inline docs/guide/homeassistant.md:238]:
```suggestion
## Manually create entities

When mqtt auto discovery and custom components aren't enough for your needs you can choose to manually create your entities in hass.

### Execute scene

If you need to create a switch that triggers a scene activation (using mqtt apis) you can use this example.

Add the following entriy in the `configuration.yaml`. Replace `<mqtt_prefix>` and `<mqtt_name>` with your values, based on your mqtt settings and `<sceneId>` with the scene you want to activate
```

File: docs/guide/homeassistant.md [hunk 3]
```diff
@@ -231,6 +231,21 @@ The main template is like the thermostat template. The things to add are:
...
+
+```yaml
+switch:
+  - platform: mqtt
+    name: Doorbell Scene
+    unique_id: zwavedoorbellscene
+    command_topic: "<mqtt_prefix>/_CLIENTS/ZWAVE_GATEWAY-<mqtt_name>/api/_activateScene/set"
+    payload_on: '{ "args": [ 1 ] }'
```
Comment (robertsLando) [inline docs/guide/homeassistant.md:246]:
```suggestion
    payload_on: '{ "args": [ <sceneId> ] }'
```

**Review by robertsLando — CHANGES_REQUESTED** [review]

**Comment by robertsLando** [comment]
@zwave-js-bot fix lint

**Pull request merged**